L1Trigger/L1TMuonBarrel/src/L1MuBMTQualPatternLut.cc

```diff
@@ -215,8 +218,24 @@ const vector<short>& L1MuBMTQualPatternLut::getQualifiedPatterns(int sp, int adr
 
   LUT::const_iterator it = m_lut.find(make_pair(sp,adr));
   if ( it == m_lut.end() ) {
-    cerr << "Error: L1MuDTQualPatternLut: no pattern list found for address " << adr << endl;
+    edm::LogError ("Not found") << "Error: L1MuDTQualPatternLut: no pattern list found for address " << adr << endl;
   }
```

Comment:
Can you be a bit more specific? Not found is a bit generic. Normally, the module name goes there.